### DISCUSSION TITLE
Add a Median() aggregate function

### DISCUSSION THREAD

**Discussion by IsThatUniqueEnough** [datalust/seq-tickets#2203] [Ideas]
Seq has a function Mean() (arithmetic average). When monitoring systems it's often desirable to have a [Median()](https://en.wikipedia.org/wiki/Median) aggregate function instead that's less affected by a few extreme values / outliers.

**Comment by nblumhardt** [comment]
Thank for the suggestion! `percentile(X, 50)` should produce what you're after; `median` would be a useful alias for this :+1:
